tests/bearlib/aspects/ExceptionsTest.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+
+from coalib.bearlib.aspects.exceptions import AspectLookupError
+
+
+class ExceptionTest:
+
+    def test_AspectLookupError(self):
+        with pytest.raises(AspectLookupError) as exc:
+            raise AspectLookupError('NOASPECT')
+        assert str(exc.value) == \
```

Comment:
please do not use line continuation ( `\` ) , as it is a parsing nightmare.  Using round brackets.